### ISSUE TITLE
Missing summed value "Verbrauch" for sensortype "universalsensor"

### ISSUE THREAD

**Issue by thetruebumblebee** [volkszaehler/volkszaehler.org#903]
as for sensor type "powersensor" is shown.
I am trying to measure the amount of rain in l/m².

Thx and Kind Regards
tom

**Comment by J-A-U** [comment]
Yes. How to compute a summary depends on the physical unit, but the whole point of universal sensor is to give you a channel without unit. With the drawback not be able to calculate consumtion.

For an alternative solution: How does your raw data look like?

**Comment by maniac-on-moon** [comment]
Thanks  for your  reply.
The  datasource  is  sending  the last  l/m2 every 300s.

**Comment by frankrichter** [comment]
You can use "consumptionsensor".

maniac-on-moon ***@***.***> schrieb am Mo., 25. Apr. 2022,
08:45:

**Comment by frankrichter** [comment]
If your datasource sends l/(m²*s) you need consumptionsensor which
integrates over time to provide consumption values.

If your datasource sends l/m² (delta since last datapoint) you need
universalcounter which only sums up the readings to get consumption values.

maniac-on-moon ***@***.***> schrieb am Mo., 25. Apr. 2022,
08:45:
